modules/infrastructure-moving-logging.adoc

```diff
@@ -3,10 +3,84 @@
 // * machine_management/creating-infrastructure-machinesets.adoc
 
 [id="infrastructure-moving-logging_{context}"]
-= Moving the logging aggregation solution
+= Moving the cluster logging resources
 
-[IMPORTANT]
+You can configure the Cluster Logging Operator to deploy the pods 
+for any or all of the Cluster Logging components, Elasticsearch, Kibana, Curator, and 
+Fluentd, to different nodes. You cannot move the Cluster Logging Operator pod from its installed location. 
+
+[NOTE]
 ====
-Cluster logging in {product-title} is not installed by default.
-You currently cannot move the logging component to another MachineSet.
+You should set your MachineSet to use at least 6 replicas.
 ====
+
+.Prerequisite
+
+* Cluster logging and Elasticsearch must be installed. These features are not installed by default.
+
+* If needed, get the name of the Cluster Logging Custom Resource in the openshift-logging project:
++
+----
+$ oc get ClusterLogging
+NAME       AGE
+instance   112m
+----
+
+.Procedure
+
+. Edit the Cluster Logging Custom Resource:
++
+----
+apiVersion: logging.openshift.io/v1
+kind: ClusterLogging
+
+....
+
+spec:
+  collection:
+    logs:
+      fluentd:
+        nodeSelector: <1>
+          infra: "yes"
+        resources: null
+      rsyslog:
+        resources: null
+      type: fluentd
+  curation:
+    curator:
+      nodeSelector: <1>
+        infra: "yes"
+      resources: null
+      schedule: 30 3 * * *
+    type: curator
+  logStore:
+    elasticsearch:
+      nodeCount: 3
+      nodeSelector: <1>
+        infra: "yes"
+      redundancyPolicy: SingleRedundancy
+      resources:
+        limits:
+          cpu: 500m
+          memory: 4Gi
+        requests:
+          cpu: 500m
+          memory: 4Gi
+      storage: {}
+    type: elasticsearch
+  managementState: Managed
+  visualization:
+    kibana:
+      nodeSelector: <1>
+        infra: "yes"
+      proxy:
+        resources: null
+      replicas: 1
+      resources: null
+    type: kibana
+
+....
+
+----
+<1> Add a `nodeSelector` parameter with the appropriate `<key>: <value>` pair to the component you want to move.
```

Comment:
Will you add an example that references the `node-role.kubernetes.io/infra: ""` `nodeSelector`?

Reply:
Is the `node-role.kubernetes.io/infra: ""` node selector required to your knowledge? I couldn't get it to work with that format. I tried the simple `tag=value` format, which was how QE tested the feature, and it worked.

Reply:
If it's not, then we should check the other node selectors.